ldap/schema/00core.ldif

```diff
@@ -305,6 +305,11 @@ attributeTypes: ( 1.3.6.1.4.1.1466.101.120.14 NAME 'supportedSASLMechanisms'
   SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
   USAGE dSAOperation
   X-ORIGIN 'RFC 4512' )
+
+attributeTypes: ( 2.16.840.1.113730.3.1.2386 NAME 'availableSASLMechanisms'
+  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
+  USAGE dSAOperation
+  X-ORIGIN 'RFC 4512' )
```

Comment:
also needs rhanana registration. X-origin may be 'RFC 4512 - related' as it is not described in 4512

Reply:
yeah I lost access to rhanana, need to open a help desk ticket.  And I will remove that RFC reference